src/bundler.ts

```diff
@@ -67,35 +61,29 @@ export async function extendBundler({ options: nuxtOptions }: I18nNuxtContext, n
   /**
    * vite plugin
    */
-
-  const vitePluginOptions: PluginOptions = {
-    allowDynamic: true,
-    runtimeOnly: nuxtOptions.bundle.runtimeOnly,
-    compositionOnly: nuxtOptions.bundle.compositionOnly,
-    fullInstall: nuxtOptions.bundle.fullInstall,
-    onlyLocales: nuxtOptions.bundle.onlyLocales,
-    dropMessageCompiler: nuxtOptions.bundle.dropMessageCompiler,
-    optimizeTranslationDirective: nuxtOptions.bundle.optimizeTranslationDirective,
-    strictMessage: nuxtOptions.compilation.strictMessage,
-    escapeHtml: nuxtOptions.compilation.escapeHtml,
-    defaultSFCLang: nuxtOptions.customBlocks.defaultSFCLang,
-    globalSFCScope: nuxtOptions.customBlocks.globalSFCScope,
-    include: localeIncludePaths
-  }
-
-  addVitePlugin(VueI18nVitePlugin(vitePluginOptions))
-  addVitePlugin(TransformMacroPlugin.vite(sourceMapOptions))
-  addVitePlugin(ResourcePlugin.vite(sourceMapOptions))
-  if (nuxtOptions.experimental.autoImportTranslationFunctions) {
-    addVitePlugin(TransformI18nFunctionPlugin.vite(sourceMapOptions))
-  }
-
+  addVitePlugin(
+    VueI18nVitePlugin({
+      ...vueI18nPluginOptions,
+      fullInstall: nuxtOptions.bundle.fullInstall,
```

Comment:
@kazupon 
These options were not set for the webpack plugin, was this intentional? 🤔

Reply:
@BobbieGoede 
Ah, I had forgotten these options for webpack. 😅 
you can specify these options for webpack.

Reply:
I see! changed it mow so all options are the same for vite and webpack 💪